code/gamemode/Game.cs

```diff
@@ -27,8 +27,8 @@ public partial class Game : Sandbox.Game
 
         public MapHandler MapHandler { get; private set; }
 
-        [ConVar.Replicated("ttt_debug")]
-        public bool Debug { get; set; } = false;
+        // [ConVar.Replicated("ttt_debug")]
+        public bool Debug { get; set; } = true;
```

Comment:
Intended as it seems that even server admin can't change this var currently ingame